dhis-2/dhis-web/dhis-web-api-test/src/test/java/org/hisp/dhis/webapi/WebClient.java

```diff
@@ -227,6 +235,12 @@ public boolean success()
             return series() == Series.SUCCESSFUL;
         }
 
+        public String contentAsString()
```

Comment:
I think it is better not to introduce this method to guide users towards the "typed" alternatives. If you need the `String` you should be able to just do `content().toString()` - this will do little extra step in-between but that should not make much of a difference. Otherwise please let me know - that would be something we should fix.

Reply:
Agree, i added that for debugging only, removed now.